modules/release-notes/partials/docs-server-7.2.4-release-note.adoc

```diff
@@ -0,0 +1,150 @@
+
+[#release-724]
+== Release 7.2.4 (January 2024)
+
+Couchbase Server 7.2.4 was released in November 2023. +
+This maintenance release contains new features and fixes.
+
+[#new-features]
+=== New Features and Enhancements
+
+* Prometheus now records information related to disk usage over time:
+* REST interface added to set the retention period of audit files
+* Erlang upgraded to version{nbsp}25.
+* Added `Memcached` alert.
+
+
+=== Cluster Manager
+
+[#table-new-features-724-cluster-manager, cols="10,40"]
+|===
+|Feature | Description
+
+| https://issues.couchbase.com/browse/MB-59237/[MB-59237]
+
+a| Prometheus now records information related to disk usage over time:
+
+* Cumulative number of bytes read from CB Data volume over time.
+* Cumulative number of bytes written to CB Data volume over time.
+* Metrics tracked in Prometheus, similar to existing system metrics.
+* Disk queue size over time (instantaneous sample of current size)
+* Disk read latency.
+
+| https://issues.couchbase.com/browse/MB-59503/[MB-59503]
+a| The request is for `ns_server` to provide a REST interface to enter the minimum duration (in minutes) that previously rotated audit files will be retained. This value will be converted to seconds and passed through to memcached. The audit daemon, run during the log rotation task, will remove any files created more than X seconds ago. +
```

Comment:
The developer added this description in the ticket:

When a query has nested correlation references, e.g., query1 contains a
subquery query2, which in turn contains another subquery query3, and
query3 is correlated with query1 (but query2 is not correlated), the
nested correlation is not properly detected and the subquery (query2)
is not properly marked as correlated.

Fix:

Correlation references are detected and marked in different levels of
subqueries, and nested correlation references are properly propagated
up such that all subqueries are properly marked when correlated.